tests/functional/spec/list_collector_primary_person.spec.js

```diff
@@ -15,7 +15,7 @@ describe('Primary Person List Collector Survey', function() {
       return helpers.openQuestionnaire('test_list_collector_primary_person.json');
     });
 
-    it('When the user says they do not live there, and changes their answer to yes, then the user can\'t navigate to the list collector', function () {
+    it.skip('When the user says they do not live there, and changes their answer to yes, then the user can\'t navigate to the list collector', function () {
```

Comment:
This test is currently skipped.

Reply:
I could remove it or leave it in for reference?. It is related to my comment here:
https://github.com/ONSdigital/eq-survey-runner/pull/2248#discussion_r309666515